src/_pytest/runner.py

```diff
@@ -339,23 +339,19 @@ def from_call(
             function, instead of being wrapped in the CallInfo.
         """
         excinfo = None
-        start = timing.time()
-        precise_start = timing.perf_counter()
+        instant = timing.Instant()
         try:
             result: TResult | None = func()
         except BaseException:
             excinfo = ExceptionInfo.from_current()
             if reraise is not None and isinstance(excinfo.value, reraise):
                 raise
             result = None
-        # use the perf counter
-        precise_stop = timing.perf_counter()
-        duration = precise_stop - precise_start
-        stop = timing.time()
+        duration = instant.duration()
         return cls(
-            start=start,
-            stop=stop,
-            duration=duration,
+            start=duration.start,
```

Comment:
We might want to make duration itself a  attribute of run result

Reply:
Not sure if I would like to expose that object as part of the public API now.